### PR TITLE
Enable flow for handling a start when running in preview

### PR DESCRIPTION
**Pull request by stulees**
## Description
Enable the flow of handling a `_start` method when running in _preview_. 

Ensuring to stop flow before the _startWatch()_ of files occur as this path of functionality is for running on Rise Player. 

## Motivation and Context
Small incremental steps to video component running in Editor Preview and Shared Schedules

## How Has This Been Tested?
Tested with template in apps https://apps.risevision.com/templates/edit/fc5e52c2-c5aa-430e-966c-a959909f9449/?cid=7fa5ee92-7deb-450b-a8d5-e5ed648c575f using Charles to map to local file of component. 

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
n/a

### CODE REVIEW

File: src/rise-video.js [hunk 1]
```diff
@@ -149,7 +150,7 @@ export default class RiseVideo extends WatchFilesMixin( ValidFilesMixin( RiseEle
       filesList = this._getDefaultFiles();
     }
 
-    const { validFiles } = this.validateFiles( filesList, VALID_FILE_TYPES );
+    const { validFiles } = super.validateFiles( filesList, VALID_FILE_TYPES );
```
Comment (olegrise) [inline src/rise-video.js:153]: @stulees curious, why this change was needed? You don't have `validateFiles` in your class, so it will find it in the base class anyway. Calling this function specifically from `super` won't make a difference. Am I missing something?

Comment (stulees) [reply src/rise-video.js:153]: @olegrise It's a personal preference change as I prefer to explicitly call `super` if there is not overriding function in the class. However, I don't feel strongly on this and can revert this back.

Comment (olegrise) [reply src/rise-video.js:153]: @stulees I see. I assume you are going to change it in other places later as well. I mean startWatch/stoptWatch for example.

**Comment by olegrise** [comment]
Looks good!

File: src/rise-video.js (hunk 1 above)
Comment (stulees) [reply src/rise-video.js]: @olegrise I reverted back, I think I'm imposing my preference on existing code, all good :)
